addons/website_slides/static/src/views/slide_channel_partner_list/slide_channel_partner_list_controller.js

```diff
@@ -0,0 +1,41 @@
+/** @odoo-module **/
+
+import { ListController } from "@web/views/list/list_controller";
+import { useService } from "@web/core/utils/hooks";
+
+
+export default class SlideChannelPartnerListController extends ListController {
+    setup() {
+        super.setup();
+        this.action = useService("action");
+        this.orm = useService("orm");
```

Comment:
Aaah, double quotes managed to make it!!!!

Reply:
Some internal discussion on that matter ? :p I see both are used in the code atm :)